appmap/test/test_django.py

```diff
@@ -116,6 +120,26 @@ def test_unnamed(client, events):
     # message attribute.
     assert len(events[0].message) == 0
 
+def test_exception(client, events):
+    with pytest.raises(RuntimeError):
+        client.get('/exception')
+
+        # The 2nd event: ExceptionEvent
+        assert events[1].items() >= {
+            'parent_id': events[0].id,
+            'exceptions': [{
+                'class': 'RuntimeError',
+                'message': 'An error'
+            }]
+        }.items()
```

Comment:
This could be moved out of the `with` block.

Reply:
We can't check `ExceptionEvent` if the `assert` block is moved out of the `with` block.

Reply:
I don't see why not. Have you tried it?

Reply:
yeah, I've tried it, but it raises `AttributeError`.
`AttributeError: 'HttpServerResponseEvent' object has no attribute 'items'`

Reply:
Ah, right, it does. But when it's in the `with` block the code just gets skipped, because `client.get` raises.

Reply:
Then I think the `HttpServerResponseEvent` should also inherit the `Exception` event to check the exception from a request.

Reply:
No. And anyway, that's beside the point. Your test just doesn't do what it's supposed to. Go ahead, remove your changes in `appmap.django`. The test still passes.